modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/main/java/com/liferay/portal/search/tuning/synonyms/web/internal/display/context/EditSynonymSetsDisplayBuilder.java

```diff
@@ -86,23 +83,29 @@ private String _getRedirect() {
 		return ParamUtil.getString(_httpServletRequest, "redirect");
 	}
 
-	private Optional<SynonymSet> _getSynonymSetOptional(long companyId) {
-		SynonymSetIndexName synonymSetIndexName =
-			_synonymSetIndexNameBuilder.getSynonymSetIndexName(companyId);
+	private SynonymSet _getSynonymSet(long companyId) {
+		String id = ParamUtil.getString(_renderRequest, "synonymSetId", null);
```

Comment:
@LisztLiszt Please rename this variable to match the string in the method call. Basically, this is to get a parameter from a render request, so the variable name should be the parameter name `synonymSetId`.